horizon/backend/db/repositories/base.py

```diff
@@ -70,10 +70,10 @@ async def _update(
         query: ReturningUpdate[tuple[Model]] = update(model_type).where(*where).values(**changes).returning(model_type)
         return await self._session.scalar(query)
 
-    async def _delete(self, id: int) -> Model | None:
+    async def _delete(self, _id: int) -> Model | None:
```

Comment:
```suggestion
    async def _delete(self, id_: int) -> Model | None:
```
`_` as prefix is used to mark variable as private. There is nothing private in method arguments.